src/core/Coins.ts

```diff
@@ -1,5 +1,5 @@
 import { fetchSupportedCoins } from "../api/api";
-import { Asset } from "./Asset";
+import type { Asset } from "./Asset";
```

Comment:
Thank you so much for this addition. Seems like a great addition to the existing patterns. Can I ask - what are these `type` imports?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html